src/test/java/htsjdk/tribble/util/ParsingUtilsTest.java

```diff
@@ -143,6 +164,18 @@ private void tstExists(String path, boolean expectExists) throws IOException{
     }
 
     @Test
```

Comment:
@droazen What do you think about including a new test dependency for providing alternative paths.  It might help us avoid problems like the index one we saw before.